documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -502,6 +502,10 @@
          "required_logs":3,
          "missing_logs":"7f8d623d0cb9966e",
          "active_generations":1,
+         "bounce_impact":{
```

Comment:
Because `bounce_impact` is intended to be a sort of generic notion that is not fundamentally tied to recoveries, it may be good to bump this out a level so that it's `cluster.bounce_impact` rather than `cluster.recovery_state.bounce_impact`.

Reply:
Done.